install-and-configure/install/cloud-integration/oracle/usage-api-cloud-cost-integration.md

```diff
@@ -0,0 +1,125 @@
+# Oracle Usage API Cloud Cost Integration
+
+This document describes how to integrate Kubecost with the Oracle Usage API to build Cloud Costs for your tenancy. This will largely be done via the OCI CLI which you will need admin privileges on.
+
+### Before Starting
+
+To begin with check ensure that you have the oracle cli installed and configured for your account, documentation on this can be found (here)[https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm#InstallingCLI__macos_homebrew]
```

Comment:
```suggestion
Ensure that you have the oracle cli installed and configured for your account. Documentation on this can be found (here)[https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm].
```